spec/support/user.rb

```diff
@@ -1,9 +1,19 @@
 class User < ActiveRecord::Base
   include ActiveStash::Search
+  include ActiveStash::Validations
   self.table_name = "users"
   self.collection_name = "activestash_test_#{ENV["ACTIVE_STASH_TEST_COLLECTION_PREFIX"] || ""}_users"
 
-  stash_index :first_name, :dob, :created_at
+  # Used for testing
+  attr_accessor :skip_validations
+
+  validates :email, uniqueness: true, if: Proc.new { |user| user.perform_validations? }
```

Comment:
```suggestion
  validates_uniqueness_of :email, if: Proc.new { |user| user.perform_validations? }
```

Finally figured out WTF is going on in Devise.  Turns out that `validates_uniqueness_of` is what Devise is calling, and that doesn't end up getting turned into a `validates uniqueness: true` call, but instead does something totally different.  NFI *what*, yet, but this at least gives us a failing test case to work from.